app/models/appropriate_body.rb

```diff
@@ -31,4 +38,8 @@ class AppropriateBody < ApplicationRecord
               message: 'Must be a number between 1000 and 9999',
               allow_blank: true
             }
+
+  def self.istip
+    find_by_name(ISTIP)
+  end
```

Comment:
This feels a little business logic-y for a method on the model. We already have `AppropriateBodies::Search`, could this just be `AppropriateBodies::Search.new.istip`?

Reply:
Sure, I am not familiar with that SO.